Tests/SonarScanner.MSBuild.PreProcessor.Test/WebClientDownloaderTest.cs

```diff
@@ -30,6 +30,32 @@ namespace SonarScanner.MSBuild.PreProcessor.Test
     [TestClass]
     public class WebClientDownloaderTest
     {
+        [TestMethod]
+        public void Ctor_SecurityProtocolIsDefault_RemainsDefault()
+        {
+            // Arrange
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.SystemDefault;
```

Comment:
is this ok to change a static field in tests? Could tests run in parallel in this case? Could some other tests be affected by this test changing these fields?

Unit tests shouldn't have side effects... https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#stub-static-references

According to https://docs.microsoft.com/en-us/dotnet/core/testing/order-unit-tests?pivots=mstest tests inside a class will run in a specific order... However you may still affect the other tests in case they rely on this property.



You should store the value at the beginning of the tests and restore it at the end of the tests